src/main/java/org/myrobotlab/opencv/ImageFileFrameGrabber.java

```diff
@@ -21,14 +24,15 @@
   private ArrayList<File> imageFiles = new ArrayList<File>();
   private int grabCount = 0;
 
-  protected transient IplImage image;
-  transient private IplImage lastImage;
+  protected transient Mat image;
+  transient private Mat lastImage;
 
   // transient private HashMap<String, IplImage> cache = new HashMap<String,
   // IplImage>();
   private int frameCounter = 0;
   public String path;
-  transient OpenCVFrameConverter.ToIplImage converter = new OpenCVFrameConverter.ToIplImage();
+  // transient OpenCVFrameConverter.ToIplImage converter = new OpenCVFrameConverter.ToIplImage();
+  OpenCVFrameConverter.ToMat converter = new OpenCVFrameConverter.ToMat();

```

Comment:
Use the new CloseableFrameConverter please and make sure it's closed in the release method.